protos/contexts/v1/contexts.proto

```diff
@@ -30,6 +30,27 @@ message Context {
     string name = 1;
 	string contextType = 2;
 	bool current = 3;
+	string description = 4;
+	oneof Endpoint {
+		DockerEndpoint docker_endpoint = 5;
```

Comment:
Is this an existing Docker context? We need to make sure that we don't confuse users. What about the local/example contexts? (Not needed now but curious what we would call them)

Reply:
Currently local contexts are not available at all in released CLI, this is the existing context. I could rename this MobyEndpoint if you prefer? this name is reflected in the client API of course, see example in the description

Reply:
Yeah, I suppose that Docker is fine for now. We can worry about naming the local context later :)